### DISCUSSION TITLE
Access from internet

### DISCUSSION THREAD

**Discussion by totrashyaru** [gerbera/gerbera#3296] [Ideas]
There is a component in the famous Foobar2000 player that uses the player's database and acts as a DLNA/UPnP server. 
Find the description of this component at https://www.foobar2000.org/components/view/foo_upnp. 

This component provides access to media library from the internet using the Foobar2000 player. I use this feature to listen to music from my home media library remotely, by using the player on smartphone or on the car head unit.

According to the description mentioned above, this feature is powered by the Platinum UPnP SDK. 

My question is, is it possible to implement the same functionality in Gerbera? I mean the access to home media library from the internet.

**Comment by KarlStraussberger** [comment]
Basically accessing everything remotely sounds interesting, but my personal safety wall blocks such an approach.

In gerbera we rely libpunp or libnpupnp and I don't know that they can provide external access via some web interface. So the request is more to those libs. Of course, we can consider building against Platinum if licenses allow and we find someone who does the initial work.
